### ISSUE TITLE
Comments highlight doesn't work properly if there's no space after the leading #

### ISSUE THREAD

**Issue by Aster89** [vim/vim#10508]
https://github.com/vim/vim/blob/b5912e0643dc21b1b88c517d3ac1282eba57293d/runtime/syntax/i3config.vim#L27

![i3config](https://user-images.githubusercontent.com/20521900/171349827-03b07bec-541c-447f-b7e0-6264211380b7.png)

**Comment by hiqua** [comment]
@Aster89 I couldn't reproduce when trying on my i3 config, can you confirm that you get that with no customization (e.g. following the steps in https://github.com/vim/vim/blob/master/CONTRIBUTING.md#reporting-issues)?

Once that's done, can you copy-paste an i3 config snippet which exhibits this bad behavior?

**Comment by Aster89** [comment]
I've reproduced it with `vim --clean`.

A trailing `{` in a comment seems to be enough to screw up syntax coloring of comments afterwards.

In the example above, you can keep lines 299 and 312-129 and see the issue:
```
# {

#bindsym $mod+Pause mode "$mode_system"

## gaps
#for_window [class=".*"] border pixel 2
#gaps inner 10
#gaps outer 0
#smart_gaps on
#smart_borders on
```

**Comment by hiqua** [comment]
Thanks, confirmed!

**Comment by jdholtz** [comment]
This has been fixed in #13054.

**Comment by chrisbra** [comment]
closing then.
